### PR TITLE
Migrated From ROS1 to ROS2

### PR DESCRIPTION
**Pull request by sandiprakhasiya**
Hi,

I have migrated pcl_msgs as per the MIgration guide and Help from some other ROS2/ROS1 source code analysis.

Thanks

### CODE REVIEW

File: package.xml [hunk 1]
```diff
@@ -16,13 +16,23 @@
   <url type="repository">https://github.com/ros-perception/pcl_msgs</url>
   <url type="bugtracker">https://github.com/ros-perception/pcl_msgs/issues</url>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  <buildtool_depend>rosidl_default_generators</buildtool_depend>
 
-  <build_depend>message_generation</build_depend>
+  <!-- build_depend>message_generation</build_depend-->
```
Comment (paulbovbel) [inline package.xml]: remove commented entries

File: package.xml [hunk 2]
```diff
@@ -16,13 +16,23 @@
...
   <build_depend>sensor_msgs</build_depend>
   <build_depend>std_msgs</build_depend>
 
-  <run_depend>message_runtime</run_depend>
-  <run_depend>sensor_msgs</run_depend>
-  <run_depend>std_msgs</run_depend>
+  <exec_depend>rosidl_default_runtime</exec_depend>
+  <exec_depend>message_runtime</exec_depend>
```
Comment (paulbovbel) [inline package.xml]: this can go as well I believe

File: CMakeLists.txt [hunk 3]
```diff
@@ -1,18 +1,42 @@
...
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+find_package(ament_cmake REQUIRED)
+# find the ROS message code generators
+find_package(rosidl_default_generators REQUIRED)
+#find_package(message_generation REQUIRED)
```
Comment (paulbovbel) [inline CMakeLists.txt]: remove commented cmake

File: CHANGELOG.rst [hunk 4]
```diff
@@ -2,6 +2,12 @@
 Changelog for package pcl_msgs
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+ROS2 (2018-12-31)
```
Comment (paulbovbel) [inline CHANGELOG.rst]: changelogs are generated at release time, please remove this

Comment (sandiprakhasiya) [reply CHANGELOG.rst]: Ok, Remove the same.

File: CMakeLists.txt (hunk 3 above)
Comment (sandiprakhasiya) [reply CMakeLists.txt]: Ok, Remove the same.

File: package.xml (hunk 1 above)
Comment (sandiprakhasiya) [reply package.xml]: Ok, Remove the same.

File: package.xml (hunk 2 above)
Comment (sandiprakhasiya) [reply package.xml]: Ohh, messge_runtime is for ROS1. 
I am removing <exec_depend>message_runtime</exec_depend>.
Thanks.

Comment (sandiprakhasiya) [reply package.xml]: Ohh, messge_runtime is for ROS1. 
I am removing <exec_depend>message_runtime</exec_depend>.
Thanks.

**Comment by sandiprakhasiya** [comment]
@paulbovbel Thanks.
Done the changes as suggested. Please review and let me know if anything else I need to do.

**Comment by paulbovbel** [comment]
Thanks, LGTM!

**Pull request merged**